regulations/templates/regulations/node.html

```diff
@@ -13,6 +18,9 @@
     {{node.marked_up|safe}}
 {% endif %}
 </p>
+{% if root.node_type == "preamble" and node.node_type == "regtext" %}
+  <div class="activate-write" data-section="{{ node.markup_id }}">Comment on paragraph in section {{ node.label_id }}</div>
+{% endif %}
```

Comment:
I may have misread, but there did seem to be `regtext` nodes nested within `preamble` nodes. Anyway, I'll verify that `not node.title` works and make the switch.